ihatemoney/static/css/download_mobile_app.css

```
@@ -0,0 +1,17 @@
+.get-it-from {
+    width: 100px;
+    min-height: 110px; 
+    border: 5px solid floralwhite;
+    background-color: white; margin: 10px;
+}
+.get-it-from:hover {
+    opacity: 80%;
+}
+main {
+    background-color: #9EDE30;
+    font-family: 'Comfortaa', arial, serif
```

```suggestion
    background-color: linear-gradient(150deg, #abe128 0%, #43ca61 100%);
    font-family: 'Comfortaa', arial, serif;
```
Use linear-gradient from home page to be consistent.